.github/workflows/master_build_merge.yml

```diff
@@ -1305,6 +1315,11 @@ jobs:
             gradlewTwineUpload
             set +e
           }
+          softwareDefBuildAndPush() {
```

Comment:
Remove this declaration and there's no `${{ steps.latest_tag.outputs.tag_ref }}`